GoogleUtilities/SwizzlerTestHelpers/GULRuntimeClassSnapshot.m

```diff
@@ -132,10 +132,9 @@ - (void)captureSelectorsAndImps {
     IMP imp = method_getImplementation(method);
     NSString *impString =
         [NSString stringWithFormat:@"%p -[%@ %@]", imp, NSStringFromClass(_aClass), methodString];
-    NSAssert(![_imps containsObject:impString],
```

Comment:
Why is it ok to remove this assert?

Reply:
@paulb777 Good question! It fails on `-[SBSStatusBarStyleOverridesAssertionData isExclusive]`. It looks like `isExclusive` is defined like both a property getter and a method ([the following header](https://github.com/nst/iOS-Runtime-Headers/blob/master/PrivateFrameworks/SpringBoardServices.framework/SBSStatusBarStyleOverridesAssertionData.h) may be a proof of it) that's why, I guess, it is detected as a duplicate. Based on it, just a condition should be fine. I could add a log message here as well, but we don't have a proper logger available in the class and I would not like to add it for the Release configuration.
WDYT?

Reply:
SGTM